instrumentation/ruby_kafka/lib/opentelemetry/instrumentation/ruby_kafka/events/consumer/process_batch.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module RubyKafka
+      module Events
+        module Consumer
+          # The ProcessBatch module contains the instrumentation for consumer process batch event
+          module ProcessBatch
+            extend self
+
+            EVENT_NAME = 'process_batch.consumer.kafka'
+            SPAN_NAME = 'kafka.consumer.process_batch'
+
+            def subscribe!
+              ::ActiveSupport::Notifications.subscribe(EVENT_NAME) do |_name, start, finish, _id, payload|
```

Comment:
This is the same problem as the `consumer.each_message` just at a coarser level.

Reply:
No longer relevant